src/components/DataModelGeneration/index.tsx

```diff
@@ -0,0 +1,125 @@
+import { Checkbox, Col, Collapse, Form, Radio, Row, Typography } from "antd";
+import { useTranslation } from "react-i18next";
+
+import TableIcon from "@/assets/table.svg";
+
+import SearchInput from "../SearchInput";
+
+import styles from "./index.module.less";
+
+import type { FC, ReactNode } from "react";
+
+const { Title, Text } = Typography;
+const { Panel } = Collapse;
+
+interface DataModelGenerationProps {
+  dataSource: {
+    icon: ReactNode;
+    name: string;
+  };
+}
+
+const items = [
+  {
+    key: "1",
+    label: "default",
+    children: (
+      <Row gutter={[8, 8]}>
+        <Col span={24}>
+          <div className={styles.field}>
+            <Checkbox>trips {"→"} Trips.js</Checkbox>
```

Comment:
tables and measures/dimensions should be provided via props

something like object:

```
tableName: {
  measures: {
   ....
  },
  dimensions: { 
  ....
  }
}
```

please check in data schemas on demo account
https://app.mlcraft.org/~/schemas/7d7eea55-ad47-4456-8a1b-badef7cf7270

demo credentials are provided in the mlcraft repo